components/basecomponents/MainCard.js

```diff
@@ -0,0 +1,6 @@
+export default function MainCard ({ children, ...props }) {
```

Comment:
If leaving an empty-ish component like this, have a more descriptive component name and drop a comment explaining what its for in the mean time. I'm not sure what MainCard by the name is and theres no code telling me what its supposed to be for.